app/managers.py

```diff
@@ -0,0 +1,38 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection =sqlite3.connect('cinema.sqlite')
```

Comment:
There is a missing space between '=' and 'sqlite3.connect'. It should be `self._connection = sqlite3.connect('cinema.sqlite')` for better readability.